python_modules/dagster-webserver/dagster_webserver/cli.py

```diff
@@ -271,7 +271,7 @@ def main():
                 os.environ[new_key] = val
 
     if sys.argv[0].endswith("dagit"):
-        WEBSERVER_LOGGER_NAME = "dagster_webserver"
+        WEBSERVER_LOGGER_NAME = "dagit"
```

Comment:
maybe regression test using  `caplog` test that would have caught this? or extend existing cli tests since click invoke gives you output

Reply:
ok, I added tests for both the dagit logger and the deprecation message